tests/rest/admin/test_user.py

```diff
@@ -2652,7 +2656,7 @@ def test_approve_account(self) -> None:
         )
 
         self.assertEqual(200, channel.code, msg=channel.json_body)
-        self.assertEqual(0, channel.json_body["approved"])
+        self.assertFalse(channel.json_body["approved"])
```

Comment:
Given that these are misnomers, I would like to second dmr's suggestion of `assertIs(.., False)` and similar for `True` to make doubly sure we're not giving 0s and 1s back. Big :/ that these test helpers are badly named.

Reply:
Sure. Though this means fixing the API to return boolean values for `admin`, `shadow_banned`, `deactivated` and `approved`, which it wouldn't do on its own (but happy to do that).